### ISSUE TITLE
Feature Request: Variable max length input for chat based on message contents

### ISSUE THREAD

**Issue by zdxn** [zdxn/ValkyriesFall#30]
Issue:
Chat currently does not have a max length set so players can type a message which is rejected.

Resolution:
```
document.querySelector("#chat-input").addEventListener("input", () => {
    let chatMessage = document.querySelector("#chat-input").value;
    if (chatMessage.indexOf("/m") > -1 && chatMessage.indexOf(":") > -1) {
        document.querySelector("#chat-input").maxLength = parseInt(chatMessage.substring(0, chatMessage.indexOf(":")).length + 160, 10);
    } else {
        document.querySelector("#chat-input").maxLength = 160;
    }
});
```

Urgency:
1
